sheetstorm/system_check.py

```diff
@@ -0,0 +1,123 @@
+import importlib  # pylint: disable=ungrouped-imports
+import sys
+from importlib import util
+from typing import Any
+
+from django.conf import settings
+from django.core.checks import Error
+from django.core.checks import Warning  # pylint: disable=redefined-builtin
+from django.core.checks import register
+
+from common.constants import PROHIBITED_SIGNS_IN_VALID_EMAIL_DOMAIN_LIST
+
+
+def create_error_01_valid_email_domain_list_is_not_a_list() -> Error:
+    return Error("The value of VALID_EMAIL_DOMAIN_LIST setting is not a list", id="sheetstorm.E001")
+
+
+def create_warning_01_duplicated_values_in_valid_email_domain_list() -> Warning:
+    return Warning(
+        "Some domains in VALID_EMAIL_DOMAIN_LIST are duplicated", hint="Remove duplicate domains", id="sheetstorm.W001"
+    )
+
+
+def create_error_02_invalid_domain_address(domain: str) -> Error:
+    return Error(
+        "Domain {} doesn't have at least one dot".format(domain),
```

Comment:
I think this one is not necessarily correct - you can set `localhost` as domain on local machine and it won't pass this check.

Reply:
I didn't think about localhost. So do you think I should add to my `if syntax` some checking for `localhost`, too ?

Reply:
I didn't think about localhost. So do you think I should and to my `if syntax` checking for `localhost` domain ?

Reply:
@rwrzesien I was wondering if `localhost` as domain is needed? I think that as a string which is pointed as `SITE_ID` - yes, but as a domain which allow users to register?

Reply:
Django adds this as a valid value to its `UrlValidator`, so I think we should accept it too - https://docs.djangoproject.com/pl/2.2/_modules/django/core/validators/ (see "localhost").